Sources/TuistKit/Commands/EnvKey/EnvKey.swift

```diff
@@ -0,0 +1,241 @@
+
+    // CLOUD ORGANIZATION REMOVE MEMBER
+    case cloudOrganizationRemoveMemberOrganizationName = "TUIST_CLOUD_ORGANIZATION_REMOVE_MEMBER_ORGANIZATION_NAME"
+    case cloudOrganizationRemoveMemberUsername = "TUIST_CLOUD_ORGANIZATION_REMOVE_MEMBER_USERNAME"
+    case cloudOrganizationRemoveMemberPath = "TUIST_CLOUD_ORGANIZATION_REMOVE_MEMBER_PATH"
+
+    // CLOUD ORGANIZATION REMOVE SSO
+    case cloudOrganizationRemoveSSOOrganizationName = "TUIST_CLOUD_ORGANIZATION_REMOVE_SSO_ORGANIZATION_NAME"
+    case cloudOrganizationRemoveSSOPath = "TUIST_CLOUD_ORGANIZATION_REMOVE_SSO_PATH"
+
+    // CLOUD ORGANIZATION UPDATE SSO
+    case cloudOrganizationUpdateSSOOrganizationName = "TUIST_CLOUD_ORGANIZATION_UPDATE_SSO_ORGANIZATION_NAME"
+    case cloudOrganizationUpdateSSOProvider = "TUIST_CLOUD_ORGANIZATION_UPDATE_SSO_PROVIDER"
+    case cloudOrganizationUpdateSSOOrganizationId = "TUIST_CLOUD_ORGANIZATION_UPDATE_SSO_ORGANIZATION_ID"
+    case cloudOrganizationUpdateSSOPath = "TUIST_CLOUD_ORGANIZATION_UPDATE_SSO_PATH"
+
+    // CLOUD PROJECT DELETE
+    case cloudProjectDeleteProject = "TUIST_CLOUD_PROJECT_DELETE_PROJECT"
+    case cloudProjectDeleteOrganization = "TUIST_CLOUD_PROJECT_DELETE_ORGANIZATION"
+    case cloudProjectDeletePath = "TUIST_CLOUD_PROJECT_DELETE_PATH"
+
+    // CLOUD PROJECT CREATE
+    case cloudProjectCreateName = "TUIST_CLOUD_PROJECT_CREATE_NAME"
+    case cloudProjectCreateOrganization = "TUIST_CLOUD_PROJECT_CREATE_ORGANIZATION"
+    case cloudProjectCreatePath = "TUIST_CLOUD_PROJECT_CREATE_PATH"
+
+    // CLOUD INIT
+    case cloudInitName = "TUIST_CLOUD_INIT_NAME"
+    case cloudInitOrganization = "TUIST_CLOUD_INIT_ORGANIZATION"
+    case cloudInitPath = "TUIST_CLOUD_INIT_PATH"
+
+    // CLOUD ORGANIZATION INVITE
+    case cloudOrganizationInviteOrganizationName = "TUIST_CLOUD_ORGANIZATION_INVITE_ORGANIZATION_NAME"
+    case cloudOrganizationInviteEmail = "TUIST_CLOUD_ORGANIZATION_INVITE_EMAIL"
+    case cloudOrganizationInvitePath = "TUIST_CLOUD_ORGANIZATION_INVITE_PATH"
+
+    // CLOUD ORGANIZATION SHOW
+    case cloudOrganizationShowOrganizationName = "TUIST_CLOUD_ORGANIZATION_SHOW_ORGANIZATION_NAME"
+    case cloudOrganizationShowJson = "TUIST_CLOUD_ORGANIZATION_SHOW_JSON"
+    case cloudOrganizationShowPath = "TUIST_CLOUD_ORGANIZATION_SHOW_PATH"
+
+    // CLOUD PROJECT LIST
+    case cloudProjectListJson = "TUIST_CLOUD_PROJECT_LIST_JSON"
+    case cloudProjectListPath = "TUIST_CLOUD_PROJECT_LIST_PATH"
+
+    // CLOUD ORGANIZATION UPDATE MEMBER
+    case cloudOrganizationUpdateMemberOrganizationName = "TUIST_CLOUD_ORGANIZATION_UPDATE_MEMBER_ORGANIZATION_NAME"
+    case cloudOrganizationUpdateMemberUsername = "TUIST_CLOUD_ORGANIZATION_UPDATE_MEMBER_USERNAME"
+    case cloudOrganizationUpdateMemberRole = "TUIST_CLOUD_ORGANIZATION_UPDATE_MEMBER_ROLE"
+    case cloudOrganizationUpdateMemberPath = "TUIST_CLOUD_ORGANIZATION_UPDATE_MEMBER_PATH"
+
+    // CLOUD AUTH
+    case cloudAuthPath = "TUIST_CLOUD_AUTH_PATH"
```

Comment:
Some flags and arguments have the same semantics across commands, like in the case of a path. What if, in those cases we reuse the environment variable? For example, we could use `TUIST_PATH` for all the `--path` options.

Reply:
It's challenging to strike a balance between granularity and simplicity. The `Test`, `Build`, and `Run` commands each have `--path` options, and there are scenarios where these should be distinct. A shared environment variable could create confusing side effects; for example, setting the build path would unintentionally set it for all commands, which might not be immediately obvious to users.

I opted for extreme explicitness as the default but am open to suggestions for a more consumer-friendly approach.

For consistency, I added an environment variable to every option, flag, and argument. However, some scenarios don't make sense for environment variables, such as `TUIST_CLOUD_ORGANIZATION_INVITE_EMAIL`. It's unlikely anyone would set that. If there's a heuristic to help developers determine what should or shouldn't be an environment variable, that would be useful. I chose to include everything to provide flexibility for consumers and reduce the cognitive load on maintainers by establishing a clear pattern.

Reply:
I think your rationale and trade-offs make total sense. Thanks for the clarification.